src/tools/dev_server/dev-server.ts

```diff
@@ -50,7 +55,10 @@ async function launch() {
   proxy.listen(server, explorePort);
   hotReloadServer.start();
 
-  console.log(`ALDS Started.\nWeb server port: ${port}\nExplorer port: ${explorePort}\nHotReload port: ${hotReloadPort}`);
+  console.log(`${GREEN}${BOLD}ALDS Started.${END}\n\nWeb server port: ${port}\nExplorer port: ${explorePort}\nHotReload port: ${hotReloadPort}`);
+  console.log(`\n${GREEN}Next, try visiting:${END}`);
+  console.log(`- http://localhost:${port}/shells/web-shell`);
+  console.log(`- http://localhost:${port}/shells/dev-shell`);
```

Comment:
This makes sense for the OSS repo, but does not for google developers.

For some time I've been pondering to have a config file for ALDS to specify things that differ between these two environments. Any other idea?

Reply:
I may be mis-remembering this, but I thought that there was a general problem with hosting ALDS within G3. 

What if we make the server a part of the CLI config and pass it to the express server? Then we can parameterize the nudge URLs. It could be localhost on local machines or a usable IP internally.

Reply:
Let's take this discussion to chat.

Reply:
1) ALDS works in google3:
blaze run third_party/javascript/arcs/src/tools/dev_server:bin

2) ALDS does more than HTTP serving, so I don't think we can use an express server instead.

As discussed offline, I was thinking that it may be enough to have a config for ALDS, that is  different in OSS and internally. e.g.

`dev-server.config`

```
{
  welcome_message: "Next, try visiting:\n- http://localhost:${port}/shells/web-shell\n- http://localhost:${port}/shells/dev-shell"
  (more stuff in the future)
}
```

If this is more than you want to sign up for that's fine.

Reply:
Better ideas welcome :)